content/en/docs/tasks/security/tls-configuration/workload-min-tls-version/index.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Istio Workload Mininum TLS Version Configuration
+description: Shows how to configure the minimum TLS version for Istio workloads.
+weight: 90
+keywords: [security,TLS]
+aliases:
+- /docs/tasks/security/workload-min-tls-version/
+owner: istio/wg-security-maintainers
+test: yes
+---
+
+This task shows how to configure the minimum TLS version for Istio workloads.
+The maximum TLS version for Istio workloads is 1.3.
+
+## Before you begin
```

Comment:
The `Before You Begin` header doesn't make sense since this session is actually showing you how to install with a minimum version. I think you can replace it with the `Configuration of minimum TLS version for Istio workloads` heading and move that paragraph up here as well.

Reply:
Done.